gatsby-node.js

```diff
@@ -1,8 +1,10 @@
 const path = require(`path`)
 const mangoSlugfy = require(`@mangocorporation/mango-slugfy`)
 const { createRemoteFileNode } = require(`gatsby-source-filesystem`)
+const { curry, __, pipe, map, isEmpty, prop, flatten } = require('ramda');
 
-const paginationPath = (page, totalPages, prefix = `/`) => {
+const paginationPath = (page, totalPages, pfx) => {
+  const prefix = isEmpty(pfx) ? `/` : pfx;
```

Comment:
Why use template strings for simple strings, I would suggest stick to single quotes when dealing with simple strings.